validator/src/main.rs

```diff
@@ -792,10 +792,11 @@ pub fn main() {
                        download from other validators"),
         )
         .arg(
-            Arg::with_name("incremental_snapshots")
-                .long("incremental-snapshots")
+            Arg::with_name("no_incremental_snapshots")
+                .long("no-incremental-snapshots")
                 .takes_value(false)
-                .long_help("Enable incremental snapshots by setting this flag. \
+                .help("Disable incremental snapshots")
+                .long_help("Disable incremental snapshots by setting this flag. \
```

Comment:
Prefer adding a new flag, hiding the old one and printing a deprecation warning when the old flag is passed. ala https://github.com/solana-labs/solana/pull/22734/files#diff-59e2068dd4d49c21ab87f3a5d8792d2daca826ab6924b32413e551c0e3035f61

Reply:
Done!